easybuild/easyblocks/a/amber.py

```diff
@@ -61,20 +63,24 @@ def extra_options(extra_vars=None):
             # enable testing by default
             'runtest': [True, "Run tests after each build", CUSTOM],
             'static': [True, "Build statically linked executables", CUSTOM],
+            'ambermpi': [False, "Build MPI versions for supported programs", CUSTOM],
         })
-        return ConfigureMake.extra_options(extra_vars)
+        return CMakeMake.extra_options(extra_vars)
 
     def __init__(self, *args, **kwargs):
         """Easyblock constructor: initialise class variables."""
         super(EB_Amber, self).__init__(*args, **kwargs)
-        self.build_in_installdir = True
+
+        if LooseVersion(self.version) < LooseVersion('19'):
```

Comment:
I think this should rather be < 20, but there is no Amber19 so if doesn't really matter.